hooks.php

```diff
@@ -53,15 +53,13 @@
 // -------------------------------------------------------------------------------------------------------------------
 
 add_filter( 'rest_index', '\Pressbooks\Api\add_help_link' );
-
 if ( $is_book ) {
 	add_action( 'rest_api_init', '\Pressbooks\Api\init_book' );
-	add_filter( 'rest_endpoints', '\Pressbooks\Api\hide_endpoints_from_book' );
 	add_filter( 'rest_url', '\Pressbooks\Api\fix_book_urls', 10, 2 );
 	add_filter( 'rest_prepare_attachment', '\Pressbooks\Api\fix_attachment', 10, 3 );
 } elseif ( $enable_network_api ) {
+	add_action( 'rest_api_init', '\Pressbooks\Api\init_book' );
```

Comment:
If API is enabled to network API, we need to allow add the endpoints for cloning including the init_book, since cloning routine now is dispatched from the main site (non-book).

Reply:
Nice finding